addons/website_sale_digital/views/website_sale_digital_view.xml

```diff
@@ -20,9 +20,26 @@
         <field name="arch" type="xml">
              <button name="toggle_active" position="before">
                 <button class="oe_stat_button" name="action_open_attachments" type="object" icon="fa-file">
-                    <field string="File" name="attachment_count" widget="statinfo" />
+                    <field string="Digital Contents" name="attachment_count" widget="statinfo" />
                 </button>
             </button>
         </field>
     </record>
+
+    <record id="ir_attachment_view_kanban_inherit_digital" model="ir.ui.view">
```

Comment:
Should be in its own ir_attachment_views file. Better classification leads to better searching.